src/app/zap-templates/zcl/data-model/chip/general-commissioning-cluster.xml

```diff
@@ -41,6 +44,16 @@ limitations under the License.
     <code>0x0030</code>
     <define>GENERAL_COMMISSIONING_CLUSTER</define>
     <description>This cluster is used to manage global aspects of the Commissioning flow.</description>
+
+    <features>
+        <feature bit="0" code="TC" name="TermsAndConditions" summary="Supports Terms and Conditions acknowledgement">
```

Comment:
The "name" here does not match the spec.  Please fix the spec.

Reply:
I created an Issue to rename to "Terms and Conditions" in the spec. https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/10170